dmapiclient/data.py

```diff
@@ -140,6 +140,12 @@ def register_framework_interest(self, supplier_id, framework_slug, user):
             user=user,
         )
 
+    def find_supplier_declarations(self, supplier_id):
+        response = self._get(
+            "/suppliers/{}/frameworks".format(supplier_id)
+        )
+        return {'declarations': response['frameworkInterest']}
```

Comment:
So I've changed it to just return `response`
Is your suggestion that returning `response['frameworkInterest']` is a better idea?